cmd/influxd/run/server_test.go

```diff
@@ -988,6 +991,16 @@ func TestServer_Query_Tags(t *testing.T) {
 			command: `SELECT value FROM db0.rp0.cpu3 WHERE company !~ /acme[23]/`,
 			exp:     `{"results":[{"series":[{"name":"cpu3","columns":["time","value"],"values":[["2012-02-28T01:03:38.703820946Z",200],["2015-02-28T01:03:36.703820946Z",100]]}]}]}`,
 		},
+		&Query{
+			name:    "single field (regex tag match with escaping)",
+			command: `SELECT value FROM db0.rp0.status_code WHERE url =~ /http\:\/\/www\.akamai\.com/`,
+			exp:     `{"results":[{"series":[{"name":"status_code","columns":["time","value"],"values":[["2015-07-22T08:13:54.929026672Z",404]]}]}]}`,
+		},
+		&Query{
+			name:    "single field (regex tag match with escaping)",
+			command: `SELECT value FROM db0.rp0.status_code WHERE url !~ /http\:\/\/www\.akamai\.com/`,
```

Comment:
There is an entry for both URLs with different values and then a test for match/no match. Might be a bit redundant.
Probably worth making the test URL influxdb.com regardless